README.md

```diff
@@ -1,2 +1,3 @@
 # training
 Documentation and workshop materials for IIIF training
+[View the gitbook](https://iiif.github.io/training/intro-to-iiif/).
```

Comment:
👍 

Would you mind adding something to effect that this is to the "Intro to IIIF workshop"? Hopefully that can be agnostic of whether or not it is a gitbook